tox.ini

```diff
@@ -0,0 +1,11 @@
+# SPDX-FileCopyrightText: 2022 Kevin Conley
+#
+# SPDX-License-Identifier: MIT
+
+[tox]
+envlist = py37
+
+[testenv]
+changedir = {toxinidir}/tests
```

Comment:
I included this `changedir` so that we run the tests from the `tests/` folder to help ensure that `adafruit_requests` gets imported from the installed package in tox's virtual environment instead of being imported locally from the root of the project folder. I believe that should result in the tests importing `adafruit_requests` in a manner more representative of how CPython users would if they installed the package via `pip`.